src/mds/Capability.cc

```diff
@@ -176,6 +181,45 @@ client_t Capability::get_client() const
   return session ? session->get_client() : client_t(-1);
 }
 
+int Capability::confirm_receipt(ceph_seq_t seq, unsigned caps) {
```

Comment:
Please reverse the order of commits. Move `::confirm_receipt` first then make changes.

Reply:
Okay, done. Thanks